test/path_matcher_test.cc

```diff
@@ -523,19 +523,24 @@ TEST_F(PathMatcherTest, CustomVerbMatches) {
 }
 
 TEST_F(PathMatcherTest, CustomVerbMatch2) {
-  MethodInfo* verb = AddGetPath("/*/*:verb");
+  MethodInfo* verb = AddGetPath("/{a=*}/{b=*}:verb");
```

Comment:
Could you also add test cases /foo/{x=**}  one with a verb and one without.

Reply:
Added